test/test-untrack.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+
+. "test/testlib.sh"
+
+begin_test "untrack"
+(
+  set -e
+
+  # no need to setup a remote repo, since this test doesn't need to push or pull
+
+  reponame="untrack"
+  git init $untrack
+  cd $untrack
+
+  # track *.jpg once
+  git lfs track "*.jpg" | grep "Tracking \*.jpg"
+  echo "* annex.backend=SHA512E" >> .gitattributes
+
+  git lfs untrack "*.jpg"
+
+  expected="* annex.backend=SHA512E"
+  [ "$expected" = "$(cat .gitattributes)" ]
+)
+end_test
+
+begin_test "untrack outside git repo"
+(
+  set -e
+
+  git lfs untrack "*.foo" || {
+    # this fails if it's run outside of a git repo using GIT_LFS_TEST_DIR
+
+    # git itself returns an exit status of 128
+    # $ git show
+    # fatal: Not a git repository (or any of the parent directories): .git
+    # $ echo "$?"
+    # 128
+
+    [ "$?" = "128" ]
+    exit 0
+  }
```

Comment:
This is failing for me locally. I put these debugging lines at the top of this test (after line 28 in the diff):

```sh
  echo "* * * * *"
  pwd
  echo "* * * * *"
  ls -al
  echo "* * * * *"
  git lfs env
  echo "* * * * *"
```

Here's the test output:

```
test: untrack outside git repo ...                                 FAILED
-- stdout --
    * * * * *
    /Users/rick/p/git-lfs-temp/test-untrack.sh-77642
    * * * * *
    total 16
    drwxr-xr-x  5 rick  staff  170 Jun 16 14:36 .
    drwxr-xr-x  4 rick  staff  136 Jun 16 14:36 ..
    drwxr-xr-x  9 rick  staff  306 Jun 16 14:36 .git
    -rw-r--r--  1 rick  staff   62 Jun 16 14:36 err
    -rw-r--r--  1 rick  staff   69 Jun 16 14:36 out
    * * * * *
    LocalWorkingDir=/Users/rick/p/git-lfs-temp/test-untrack.sh-77642
    LocalGitDir=/Users/rick/p/git-lfs-temp/test-untrack.sh-77642/.git
    LocalMediaDir=/Users/rick/p/git-lfs-temp/test-untrack.sh-77642/.git/lfs/objects
    TempDir=/Users/rick/p/git-lfs-temp/test-untrack.sh-77642/.git/lfs/tmp
    ConcurrentTransfers=3
    BatchTransfer=false
    GIT_LFS_TEST_DIR=/Users/rick/p/git-lfs-temp
    GIT_LFS_TEST_MAXPROCS=4
    * * * * *
    GIT_LFS_TEST_DIR should be set outside of any Git repository
```

It's still finding the local git dir as "LocalGitDir=/Users/rick/p/git-lfs-temp/test-untrack.sh-77642/.git", which actually seems to exist. I'm not sure what's creating it though.